gtfs/spec/en/reference.md

```diff
@@ -198,23 +199,23 @@ File: **Required**
+|  arrival_time | **Required** | The **arrival_time** specifies the arrival time (always in agency_timezone) at a specific stop for a specific trip on a route. The time is measured from "noon minus 12h" (effectively midnight, except for days on which daylight savings time changes occur) at the beginning of the service day in the timezone of the corresponding agency for the trip. For times occurring after midnight on the service day, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. If you don't have separate times for arrival and departure at a stop, enter the same value for **arrival_time** and **departure_time**.<br/><br/>Scheduled stops where the vehicle strictly adheres to the specified arrival and departure times are timepoints. For example, if a transit vehicle arrives at a stop before the scheduled departure time, it will hold until the departure time. If this stop is not a timepoint, use either an empty string value for the **arrival_time** field or provide an interpolated time. Further, indicate that interpolated times are provided via the **timepoint** field with a value of zero. If interpolated times are indicated with **timepoint**=0, then time points must be indicated with a value of 1 for the **timepoint** field. Provide arrival times for all stops that are time points.<br/><br/>An arrival time must be specified for the first and the last stop in a trip. Times must be eight digits in HH:MM:SS format (H:MM:SS is also accepted, if the hour begins with 0). Do not pad times with spaces. The following columns list stop times for a trip and the proper way to express those times in the **arrival_time** field: |  |
```

Comment:
`An arrival time must be specified for the first and the last stop in a trip.`

A) This is redundant as it's a mandatory field
B) How the first stop of a trip have an arrival_time in most cases it will only have a departure_time.

Reply:
Quoting @linusnorton 

> ```An arrival time must be specified for the first and the last stop in a trip.```
> 
> A) This is redundant as it's a mandatory field
> B) How the first stop of a trip have an arrival_time in most cases it will only have a departure_time.

The Spec has long required the first and last stops to have arrival and departure times. Several consuming apps and validation tools require this. This has always struck me as odd, and it precipitates other issues, like block overlap. See: https://groups.google.com/d/msg/gtfs-changes/u1UuL1VNVSk/W0CdAPN0DgAJ

I'd support a change here but represent just one organization. I think that there are significant issues to consider and hurdles to overcome w.r.t. backwards compatibility.